### PR TITLE
Nerfs blobs

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Normal blob spores now take two welder hits to die, from three.
tweak: Cyborgs now die much less rapidly to blob attacks.
/:cl:

All mobs now return QDEL_HINT_HARDDEL because oh wow maybe that was why overminds were in nullspace for a while

### CODE REVIEW

**Comment by Chocobro** [comment]
seems ok :+1:

File: code/game/gamemodes/blob/blobs/blob_mobs.dm [hunk 1]
```diff
@@ -82,16 +82,16 @@
...
+	maxHealth = 30
 	verb_say = "psychically pulses"
 	verb_ask = "psychically probes"
 	verb_exclaim = "psychically yells"
 	verb_yell = "psychically screams"
 	melee_damage_lower = 2
 	melee_damage_upper = 4
 	obj_damage = 20
-	environment_smash = 0
+	environment_smash = 1
```
Comment (phil235) [inline code/game/gamemodes/blob/blobs/blob_mobs.dm]: this won't change anything mechanically, the spore will still take multiple attacks to break tables and stuff. The only thing that environment_smash = 1 brings is the big smash impact sound.

Comment (ChangelingRain) [inline code/game/gamemodes/blob/blobs/blob_mobs.dm]: Phil, look at `/mob/living/simple_animal/hostile/proc/DestroySurroundings()`

Comment (phil235) [inline code/game/gamemodes/blob/blobs/blob_mobs.dm]: Oh, I see. Makes sense. Although it'll be weird for blob sport to do the big impact noise.

**Pull request merged**